ShapesDemo.pro

```diff
@@ -124,3 +121,17 @@ RESOURCES += \
     images/eprosimalogo.qrc
 
 RC_FILE = images/eprosima_icon.rc
+
+win32:CONFIG(release, debug|release): LIBS += -L$$PWD/../../Fast-RTPS/allocation-qos-release/build/install/lib/release/ -lfastrtps
```

Comment:
These lines seem to be dependent on your build. They shouldn't be necessary, as ShapesDemo is supposed to be built using an installed version of Fast-RTPS. That is why we added the colcon.pgk file

Reply:
You are totally right. Removing them.